lib/util.js

```diff
@@ -71,10 +71,13 @@ const errorToString = Error.prototype.toString;
 var CIRCULAR_ERROR_MESSAGE;
 var Debug;
 
-// eslint-disable-next-line
-const escapeSequencesRegExp = /[\\'\x00-\x1f]/g;
+/* eslint-disable */
+const strEscapeSequencesRegExp = /[\x00-\x1f\x27\x5c]/;
+const keyEscapeSequencesRegExp = /[\x00-\x1f\x27]/;
+const strEscapeSequencesReplacer = /[\x00-\x1f\x27\x5c]/g;
+const keyEscapeSequencesReplacer = /[\x00-\x1f\x27]/g;
```

Comment:
Is there really a performance difference between the 'g' modifier and without with `.test()`? If not, we shouldn't duplicate these regexps.

Reply:
const re = /test/g

console.log(re.test("tester")) // true
console.log(re.test("tester")) // false